From: Matt Benjamin <mbenjamin@redhat.com>
Date: Wed, 17 Aug 2016 10:52:58 -0400
Subject: rgw_file: add RGW_OPEN_FLAG_STATELESS as a synonym for V3

Signed-off-by: Matt Benjamin <mbenjamin@redhat.com>
(cherry picked from commit da596eb3af69c2cdf191893eb44a8947662634cf)

Resolves (with following): rhbz#1370331
---
 src/include/rados/rgw_file.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/include/rados/rgw_file.h b/src/include/rados/rgw_file.h
index b8545be..a1ef37e 100644
--- a/src/include/rados/rgw_file.h
+++ b/src/include/rados/rgw_file.h
@@ -222,6 +222,7 @@ int rgw_truncate(struct rgw_fs *rgw_fs,
 #define RGW_OPEN_FLAG_NONE         0x0000
 #define RGW_OPEN_FLAG_CREATE       0x0001
 #define RGW_OPEN_FLAG_V3           0x0002 /* ops have v3 semantics */
+#define RGW_OPEN_FLAG_STATELESS    0x0002 /* alias it */
 
 int rgw_open(struct rgw_fs *rgw_fs, struct rgw_file_handle *parent_fh,
 	     uint32_t posix_flags, uint32_t flags);
